Source/EventFlow/Subscribers/DispatchToEventSubscribers.cs

```diff
@@ -114,10 +118,28 @@ private class SubscriberInfomation
                 {
                     await subscriberInfomation.HandleMethod(subscriber, domainEvent, cancellationToken).ConfigureAwait(false);
                 }
-                catch (Exception e) when (swallowException)
+                catch (Exception e)
                 {
-                    _log.Error(e, $"Subscriber '{subscriberInfomation.SubscriberType.PrettyPrint()}' threw " +
-                                  $"'{e.GetType().PrettyPrint()}' while handling '{domainEvent.EventType.PrettyPrint()}': {e.Message}");
+                    var handled = await _recoveryHandlerProcessor.RecoverSubscriberErrorAsync(
+                            subscriber,
+                            domainEvent,
+                            e,
+                            cancellationToken)
+                        .ConfigureAwait(false);
+
+                    if (handled)
+                    {
+                        continue;
+                    }
+
+                    if (swallowException)
```

Comment:
Intersting idea: swallow exception could be implemented just by installing NopRecoverHandler for them.

Reply:
Removed for now